lib/io/pwm.cpp

```diff
@@ -0,0 +1,71 @@
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "pwm.hpp"
+
+namespace hyped::io {
+
+Pwm::Pwm(const uint8_t channel, hyped::core::ILogger &log) : log_(log)
+{
+  log_.log(hyped::core::LogLevel::kInfo, "Pwm initiated on channel %d", channel);
+  path_ = "/sys/class/pwm/pwmchip" + std::to_string(channel) + "/pwm" + std::to_string(channel);
```

Comment:
is this absolutely static and unlikely to change or should it be in a config file somewhere?